src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './App.css';
+import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <Calculator />
+  </React.StrictMode>,
```

Comment:
- [ ] Great job but you should have the `App.js` file which is the root component between Calculator components and the entry point of the app (`index.js`) so that all your components will be nested in the App.js component instead of being directly imported into the entry point and only the root component (`App.js`) that includes all the components will be imported into the index.js. I hope that makes sense.:nerd_face::innocent:

- It should be like this:

```
import Name from ./components; 

class App extends Component {
  render() {
    return {
      <Namer />
    };
  }
}
```

- I know you can do it 😄 💪🏻